01-introduction.Rmd

```diff
@@ -2,68 +2,65 @@
 
 **Abstract**
 
-This chapter welcomes readers and gives an overview of the book, its themes, and its structure. It walks through what it means to learn about data science in education and the complex nature of education systems. The chapter describes how we clarify the data scientist role through a series of walkthroughs that introduce foundational skills for those new to learning R, run analysis on student perceptions of learning and student performance data, and describe how to use aggregate data.
-
+This chapter is a welcome to readers and an overview of the book. It describes challenges with learning data science in education, including the complexity of the data scientist role. Finally, this chapter introduces data analysis walkthroughs as a way to learn data science in the education field.
```

Comment:
Super minor - should we change "data analysis walkthroughs" to "walkthroughs"? Reasoning: data analysis != data science, and this is simpler